### PR TITLE
💎(site) Add new design to Primary Navigation

### LINKED ISSUE #260: Module: Primary Navigation
Design: https://www.figma.com/file/xtBKZcRBo7H2Jrjs7DjNqE/Final-Designs?node-id=2%3A4714

### PR DESCRIPTION
**Pull request by Vanss472**
The navigation was updated with new styling and markup.

Notes from design and dev:
1. To be consistent with other Google sites, we have switched from the arrow-drop-down to a **caret**.
2. Also, the **Subscribe** CTA has been moved to the mobile menu footer.
3. Expanded search bar on mobile

- Resolves #260

### CODE REVIEW

File: site/sass/components/_nav.scss [hunk 1]
```diff
@@ -257,7 +238,7 @@
 
   &__primary-item {
     @include minLangMQ($bkpt-map--header--large) {
-      padding-bottom: 1rem;
+      padding-bottom: 1.18rem;
```
Comment (Snugug) [inline site/sass/components/_nav.scss:241]: This number seems off to me; it doesn't align to a `px` value (it's `18.88px`) and it doesn't align to other scales we have. Where did it come from? Can we use `1.25rem` instead?

**Review by Snugug — CHANGES_REQUESTED** [review]
Thanks so much for your work on this! It all looks great! I just have one question about one value, otherwise this looks GTG

File: site/sass/components/_nav.scss (hunk 1 above)
Comment (Vanss472) [reply site/sass/components/_nav.scss:241]: It actually comes from `19px` but yes I can change this value to `1.25rem`

Comment (Vanss472) [reply site/sass/components/_nav.scss]: Fixed.

**Review by Snugug — APPROVED** [review]

**Pull request merged**